src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationSdkHelper.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="EnvironmentConfigurationSdkHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using OpenTelemetry.Context.Propagation;
+
+namespace OpenTelemetry.AutoInstrumentation.Configuration;
+
+internal static class EnvironmentConfigurationSdkHelper
+{
+    public static void UseEnvironmentVariables(SdkSettings settings)
+    {
+        SetDefaultTextMapPropagator(settings.Propagators);
+    }
+
+    private static void SetDefaultTextMapPropagator(IList<Propagator> settingsPropagators)
+    {
+        if (settingsPropagators.Count == 0)
+        {
+            return;
```

Comment:
nit: a comment here saying that in this is case it simply uses the Sdk defaults seems good.

Reply:
Will be fixes in follow up PR.